### PR TITLE
STAGING changes for introducing OneGet

### PR DESCRIPTION
**Pull request by mylesbkeating1993**
I'm not as familiar with the Nano flow so a few questions:
1. Will the Docker client be installed on Nano? Would it even function?
2. Is there anything peculiar to remote PS management I didn't call out?

### CODE REVIEW

**Comment by mylesbkeating1993** [comment]
@enderb-ms

**Comment by neilpeterson** [comment]
Based on my experience here, this looks correct to me. I however have not tested it.

**Comment by mylesbkeating1993** [comment]
OK I'll test once we publish

File: virtualization/windowscontainers/deployment/deployment_nano.md [hunk 1]
```diff
@@ -58,64 +58,34 @@ Restart-Computer
...
-dockerd --register-service
+Restart-Computer -Force
 ```
 
-Start the Docker service.
+Once the server reboot is complete, you can re-establish the remote connection and start the Docker service.
 
 ```none
-Start-Service Docker
+Start-Service docker
```
Comment (mylesbkeating1993) [inline virtualization/windowscontainers/deployment/deployment_nano.md]: I'm not sure, but I trust you guys and I'll confirm when we publish

**Pull request merged**